playground/mocks/idp/idx/data/identify-with-no-sso-extension.json

```diff
@@ -0,0 +1,61 @@
+{
+  "stateHandle": "02IAickZV0j0TL0JaJJUxe4GnX7YHTeZY9pCYe-pBZ",
+  "version": "1.0.0",
+  "expiresAt": "2020-15T17:28:11.000Z",
+  "step": "IDENTIFY",
+  "intent": "LOGIN",
+  "remediation": {
+      "type": "array",
+      "value": [
+          {
+                "rel": ["create-form"],
+                "name": "cancel-transaction",
```

Comment:
what is the difference between this form with name `cancel-transaction` and the `cancel` action that you get in the response ?

Reply:
this form name should actually be more specific, since it's sort of globally unique, something like `cancel-sso-extension`. Would need a backend change as well.

Reply:
The original idea is to make it to cancel authenticator transaction. So maybe we can rename it as cancel-authenticator-transaction

Reply:
@royalchan-okta my reasoning behind naming it as `cancel-transaction` is to keep it generic so that it could be used for any future use cases where we want to cancel transactions like canceling any future authenticator/identification transactions etc. Do you think `transaction` is kind of an internal detail that SIW need not have to know and is that the reason to make it specific?

Reply:
Hm, that's actually a good point, we could retrofit this form to anything by changing the url. Alright, I'm convinced we can leave it this way for now as long as there isn't issues with the SIW views

Reply:
Thanks for the feedback. Will keep the remediation name as it is.